### PR TITLE
Sprint Submission question removal

### PR DESCRIPTION
**Pull request by mellogarrett**
## Jira Issue

[CORE-1425]

## Description

- Remove questions from Sprint Submission Form. 
- Update `atlas-db` 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

[CORE-1425]: https://lambdaschool.atlassian.net/browse/CORE-1425

### CODE REVIEW

**Comment by boblauer** [comment]
Did you make sure the question title & descriptions are what was listed in the Jira issue?

**Comment by mellogarrett** [comment]
No, I thought I was just adjusting the order.

**Comment by boblauer** [comment]
Cross reference the title and descriptions as well, not sure if there were changes there but if so I want to make sure we capture them.

File: forms/sprint-retrospective.ts [hunk 1]
```diff
@@ -29,8 +15,7 @@ export const SprintSubmissionForm: Form = {
       key: 'studentId',
     },
     {
-      title: 'What did you study during this sprint?',
-      description: `If you are currently in Labs or CS, look for your original section with this week's sprint name. For example, a WEB20 Student in "CS19 Graphs" would look for "WEB20 Graphs".`,
```
Comment (MarkDKoslow) [inline forms/sprint-retrospective.ts]: Surprised we're removing this 🤔

Comment (boblauer) [reply forms/sprint-retrospective.ts]: It's worth running these deletes by Joshua in the Jira issue to verify that he purposefully left them out and it wasn't an oversight.

Comment (mellogarrett) [reply forms/sprint-retrospective.ts]: Will do

Comment (mellogarrett) [reply forms/sprint-retrospective.ts]: Joshua said it's been removed because this field will be auto-populated for them.

Comment (MarkDKoslow) [reply forms/sprint-retrospective.ts]: It won't auto-populate in a lot of cases. I can talk to Joshua but I would leave it. I've seen students get confused by this, bc when they're in CS/Labs, they think their section is CS and not their original section.

**Comment by MarkDKoslow** [comment]
Looks good ✅ Don't think that description question should block launch

**Pull request merged**